JsonClient/JsonClient/Program.cs

```diff
@@ -23,7 +24,7 @@ static async Task runClient()
              while(option < 7)
              {
                 Console.WriteLine("Select an entity option:\n1 Comments\n2 Post\n3 Album\n4 Photo\n5 Todo\n6 User\n7 exit");
-                option = Int32.Parse(Console.ReadLine());
+                int.TryParse(Console.ReadLine(), out option);

```

Comment:
What happens if the user just hit `Enter` key? Review also the case in CrudMenu method.